gef.py

```diff
@@ -6410,7 +6411,6 @@ def context_code(self):
         arch_name = arch.name().lower()
         if is_arm_thumb():
             arch_name += ":thumb"
-            pc   += 1
```

Comment:
I assume this was meant as a fix for context code.

Reply:
Yeah, this is just to set the lsb to 1. Why do you want to remove it?

Reply:
It gives the wrong addresses in the code context, why do you need to set the lsb? I don't how it is used in `gef_disassemble`? Only seems incorrect to call `gdb_get_nth_previous_instruction_address` with something other than the correct `pc`. Can you explain that?